components/lcd/esp_lcd_sh1107/README.md

```diff
@@ -0,0 +1,98 @@
+# ESP LCD SH1107
+
+Implementation of the SH1107 LCD controller with esp_lcd component. 
+
+| LCD controller | Communication interface | Component name | Link to datasheet |
+| :------------: | :---------------------: | :------------: | :---------------: |
+| SH1107         | I2C                     | esp_lcd_sh1107 | [WIKI](https://www.waveshare.com/wiki/1.3inch_OLED_Module_(C)) |
+
+## Add to project
+
+Packages from this repository are uploaded to [Espressif's component service](https://components.espressif.com/).
+You can add them to your project via `idf.py add-dependancy`, e.g. 
+```
+    idf.py add-dependency esp_lcd_sh1107==1.0.0
+```
+
+Alternatively, you can create `idf_component.yml`. More is in [Espressif's documentation](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/tools/idf-component-manager.html).
+
+## Usage
+
+For detailed usage, please go to [LCD documentation](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/peripherals/lcd.html).
+
+## Example initialization 
+
+```
+#define LCD_SH1107_I2C_ADDRESS  0x3C
```

Comment:
Could we define SH1107 address in `esp_lcd_sh1107.h` ?

Or maybe even better, define a helper macro that would initialize `const esp_lcd_panel_io_i2c_config_t io_config = SOME_SH1107_INIT_MACRO()` ?